executor/update.go

```diff
@@ -41,32 +41,24 @@ type UpdateExec struct {
 	updatedRowKeys map[int64]map[int64]bool
 	tblID2table    map[int64]table.Table
 
-	rows        [][]types.Datum // The rows fetched from TableExec.
```

Comment:
rows and newRowsData will be written into binlog, can we remove it safely here?

Reply:
I'm not clear about it. But I don't see where these two fields are used elsewhere and the compiler doesn't tell me anything wrong...

Reply:
It's irrelevant here.
Binlog will be collected in 2PC via memdb (the dest of the write) data.